simtools/job_submission/job_manager.py

```diff
@@ -0,0 +1,207 @@
+import logging
+import os
+from copy import copy
+
+import simtools.util.general as gen
+
+__all__ = ["JobManager"]
+
+class MissingWorkloadManager(Exception):
+    pass
+
+class JobManager:
+    """
+    JobManager provides an interface to workload managers
+    like gridengine or HTCondor.
+
+    Attributes
+    ----------
+    submitCommand: str
+       job submission command (allowed is qsub, condor_submit, seriell_script
+    test: bool
+       testing mode without sub submission
+
+    Methods
+    -------
+    test_submission_system
+       Check that the requested workload manager exis
+    submit(run_script, run_out_file)
+       Submit a job described by a shell script
+
+    """
+
+    def __init__(
+        self,
+        submitCommand=None,
+        test=False
+    ):
+        """
+        JobManager init
+
+        Parameters
+        ----------
+        submitCommand: str
```

Comment:
What is expected as the submit command here for example? A full qsub command with ``log_out`` etc.? I think that's what I understood from below, but I am not sure. If so, shouldn't we build it here like it's done for hcondor?

Reply:
Agree - changed it to use the keyword `qsub` only and write always to out and err. Not sure why Raul had the option of specifying this in the config file. I think having a consistent way of doing things between qsub and HTcondor is good.